web_src/js/webcomponents/overflow-menu.js

```diff
@@ -22,18 +23,34 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
               items[0].focus();
             }
           }
-        } else if (e.code === 'Escape') {
+        } else if (e.key === 'Escape') {
           e.preventDefault();
+          e.stopPropagation();
           this.button._tippy.hide();
           this.button.focus();
-        } else if (e.code === 'Space') {
-          // eslint-disable-next-line unicorn/no-lonely-if
-          if (document.activeElement.matches('[role="menuitem"]')) {
-            e.preventDefault();
+        } else if (e.key === ' ' || e.code === 'Enter') {
+          e.preventDefault();
+          e.stopPropagation();
```

Comment:
It shouldn't "prevent" too early, for example, maybe there will be "input" in the popup

Reply:
Ah, I was replicating from [here](https://github.com/tailwindlabs/headlessui/blob/626a253dcf09589e2a2fbbfd0c279120153394cc/packages/%40headlessui-vue/src/components/menu/menu.ts#L257), but I see it's likely better to do conditionally.

Reply:
fixed with https://github.com/go-gitea/gitea/pull/29400/commits/97ed9ed33f871d36c5885f50ccc868d5fe6383bf